templates/testutil/prompt.go

```diff
@@ -104,7 +104,7 @@ func DialogTest(ctx context.Context, tb testing.TB, steps []DialogStep, cmd cli.
 	}()
 
 	select {
-	case <-time.After(time.Second): // time is arbitrary
+	case <-time.After(5 * time.Second): // time is arbitrary
```

Comment:
Wondering should we make such time second value a parameter instead of a value that should satisfy all callers?

Reply:
I think that's unnecessary. The callers don't need to be different, and that just adds complexity to using this feature, whose whole purpose is convenience. If we were exposing this as a public API outside of the abc project, I think this would be worth doing, but not as it currently is.

Also, this timeout only happens on test failures, not on successes, so it's not really wasting anyone's time.